### ISSUE TITLE
Standardized coefficients for simple slopes analyses

### ISSUE THREAD

**Issue by IRoss10** [gamlj/gamlj#167]
Hi there,

I'm using the GAMLj package (general linear model) to conduct simple slopes analysis and would like to include the effect sizes for the simple slopes of my interaction.

Is it possible to get the standardized coefficients somewhere in here or another effect size? 

![image](https://user-images.githubusercontent.com/82854588/140864614-1cecc481-db07-47ad-8fa5-ba0a6f0b7aa9.png)

**Comment by mcfanda** [comment]
Hi, Version 2.6.1, which came out last week, features both betas and effect size index for simple effects in GLM

**Comment by IRoss10** [comment]
Hi there, does that also provide betas/ effect sizes for logistic regression?

**Comment by mcfanda** [comment]
In logistic regression, it is not clear how to define "betas", because the estimated values are logits. You can simply get coefficients in a sort of standardized version by standardizing your independent variables (Covariate scaling). Thus, you obtain the change in logits (estimate) and in odd (expB) for one standard deviation more in the independent variable. This is what many would consider as the standardized version of the logistic coefficient.
